tools/toolchains/mbed_toolchain.py

```diff
@@ -110,6 +110,7 @@
 }
 
 UNSUPPORTED_C_LIB_EXCEPTION_STRING = "{} C library option not supported for this target."
+UNSUPPORTED_TOOLCHAIN_EXCEPTION_STRING = "{} toolchain does not have default c library for this target."
```

Comment:
You don't need a new message. The UNSUPPORTED_C_LIB_EXCEPTION_STRING message is correct and can be re-used.